app/src/main/java/com/android/doctorapp/ui/doctor/AddDoctorViewModel.kt

```diff
@@ -414,6 +424,137 @@ class AddDoctorViewModel @Inject constructor(
         }
     }
 
+    fun test() {
+        viewModelScope.launch {
+            /*when (val response = authRepository.testGeoHash(fireStore)) {
+                else -> {
+
+                }
+            }*/
+
+            val latitude = 23.0225
+            val longitude = 72.5714
+
+            val geohash = GeoFireUtils.getGeoHashForLocation(GeoLocation(latitude, longitude))
```

Comment:
Please remove unused variable and remove all commented code in this task

Reply:
Done